From ed6dc71727c7a0b5df43914f317885ac6bbda040 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Fri, 2 Nov 2018 14:22:58 +0800
Subject: [PATCH 4997/5242] MLK-20177-2 arm64: dts: imx8mm: use dedicate
 compatible string for usb

commit  b7a928d8b14ca8c0dd5660d0c24fbc765b27c39e from
https://source.codeaurora.org/external/imx/linux-imx.git

Use "fsl, imx8mm-usb" for imx8mm usb controller driver.

Acked-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
index 285775a..e19ff87 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
@@ -607,7 +607,7 @@
 	};
 
 	usbotg1: usb@32e40000 {
-		compatible = "fsl,imx7d-usb", "fsl,imx27-usb";
+		compatible = "fsl,imx8mm-usb", "fsl,imx7d-usb", "fsl,imx27-usb";
 		reg = <0x0 0x32e40000 0x0 0x200>;
 		interrupts = <GIC_SPI 40 IRQ_TYPE_LEVEL_HIGH>;
 		clocks = <&clk IMX8MM_CLK_USB1_CTRL_ROOT>;
@@ -637,7 +637,7 @@
 	};
 
 	usbotg2: usb@32e50000 {
-		compatible = "fsl,imx7d-usb", "fsl,imx27-usb";
+		compatible = "fsl,imx8mm-usb", "fsl,imx7d-usb", "fsl,imx27-usb";
 		reg = <0x0 0x32e50000 0x0 0x200>;
 		interrupts = <GIC_SPI 41 IRQ_TYPE_LEVEL_HIGH>;
 		clocks = <&clk IMX8MM_CLK_USB1_CTRL_ROOT>;
-- 
1.7.9.5

